### ISSUE TITLE
With `--engine lightpanda` snapshotting fails

### ISSUE THREAD

**Issue by haarts** [vercel-labs/agent-browser#780]
I'm trying a different engine, namely lightpanda and am following: https://agent-browser.dev/engines/lightpanda

The command `agent-browser --engine lightpanda snapshot -i` fails with: `✗ Failed to deserialize CDP response for Accessibility.getFullAXTree: invalid type: integer `2`, expected a string`.

My version of Lightpanda is: fe3faa0a
Agent-browser version: 0.19.0

PS the docs referenced earlier references the `agent-browser --engine lightpanda screenshot`. That *can't* work since LightPanda doesn't render pixels.

**Comment by ctate** [comment]
@haarts Fixed in #775 and just published in 0.20.1

**Comment by haarts** [comment]
Hahaha! That's awesome.
